src/main/groovy/edu/oregonstate/mist/personsapi/mapper/PersonMapper.groovy

```diff
@@ -0,0 +1,33 @@
+package edu.oregonstate.mist.personsapi.mapper
+
+import edu.oregonstate.mist.core.PersonObject
+import edu.oregonstate.mist.personsapi.PhoneFormatter
+import org.skife.jdbi.v2.StatementContext
+import org.skife.jdbi.v2.tweak.ResultSetMapper
+
+import java.sql.ResultSet
+import java.sql.SQLException
+
+public class PersonMapper implements ResultSetMapper<PersonObject> {
+    PhoneFormatter phoneFormatter = new PhoneFormatter()
+    public PersonObject map(int i, ResultSet rs, StatementContext sc) throws SQLException {
+        new PersonObject (
+            osuID: rs.getString('OSU_ID'),
+            firstName: rs.getString('FIRST_NAME'),
+            lastName: rs.getString('LAST_NAME'),
+            middleName: rs.getString('MIDDLE_NAME'),
+            birthDate: rs.getDate('BIRTH_DATE'),
+            primaryAffiliation: rs.getString('PRIMARY_AFFILIATION'),
+            email: rs.getString('EMAIL_ADDRESS'),
+            username: rs.getString('USERNAME'),
+            osuUID: rs.getString('OSUUID'),
+            confidential: rs.getBoolean('CONFIDENTIAL'),
```

Comment:
For all these `getBoolean` calls, be careful because it looks like that method only returns true if the column contains a 1: https://docs.oracle.com/javase/7/docs/api/java/sql/ResultSet.html#getBoolean(java.lang.String)

Since boolean/ind fields in banner are often represented by `Y` or `null`, or `Y` or `N`, it might be better to explicitly check for a certain value instead of using `getBoolean`. 

As an example, here is what I did for the students API: https://github.com/osu-mist/students-api/commit/a765a49a10e5b984fdd769005b6606635ac2952e#diff-b4c2ee6fa8bbe8377a138dd3912be159R14

Reply:
For integration tests, we'll want to have a test to check that a person we know is confidential in banner is being represented as confidential in our API.

Reply:
Good catch, thanks. I mapped the `currentStudent` field by calling `getString` and check if the field is equal to `Y`. However, I didn't change `confidential` and `currentUser` mapping methods since these indicators using `1` and `0` here. Do you think I should also explicitly check these two filed even they are already convertible?

Reply:
We discussed this in person, but to summarize, I don't think we need to explicitly check unless the column is something other than 1 or null, or 1 or 0, since those values would correctly map to true or false in the getBoolean method.